Fix Handler InventoryOperationResponse

sakipvp opened the pull request.

SDClowen commented:
You're just changing the architecture and saying it's a fix. Try using this word if you've actually fixed a bug! Also your codebase is not suitable for **RSBot**.
`AbilityPet: IBag`
It looks like bad architecture. Instead, you could make a common **Inventory** class.

```cs
class AbilityPet{
    public Inventory Inventory;
}
```

SDClowen commented:
Also No need to fork every time. You can update your existing fork
Visual Studio Menu > Open in command prompt 
https://www.youtube.com/watch?v=sVVXHX0Ec9Q

sakipvp commented:
After lots of tests and parse packets I saw there was no one of these operations, what worked full correctly.
So it is not a simply arhitecture change, there were lots of bugs in all these operations. I just know to hope I fixed all.
The IBag was an idea(the fastest), but there is no #development channnel on Discord, where I could have shared it before I implemented it.
-
About fork: You are right.
